### DISCUSSION TITLE
Daily build: blazor.boot.json not found.

### DISCUSSION THREAD

**Discussion by marinasundstrom** [dotnet/aspnetcore#48879] [Q&A] [answered]
I'm trying out Blazor SSR with WebAssembly in a daily build - with an issue.

When I load the WebAssembly component the ``/blazor.web.js`` tries to load ``/_framework/blazor.boot.json`` but it doesn't exist.

What am I missing?

Daily build of SDK version: 8.0.100-preview

![Screenshot 2023-06-18 at 22 04 05](https://github.com/dotnet/aspnetcore/assets/919580/4a81f01f-4644-45b8-8e8f-bb99a34a3857)
.6.23316.32

```xml
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>net8.0</TargetFramework>
    <Nullable>enable</Nullable>
    <ImplicitUsings>enable</ImplicitUsings>
  </PropertyGroup>

  <PropertyGroup>
    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.Server" Version="8.0.0-preview.6.23317.1" />
  </ItemGroup>

</Project>
```

```c#
using StoreSite;

var builder = WebApplication.CreateBuilder(args);

// Add services to the container.
builder.Services.AddRazorComponents()
                .AddServerComponents()
                .AddWebAssemblyComponents();

var app = builder.Build();

// Configure the HTTP request pipeline.
if (!app.Environment.IsDevelopment())
{
    app.UseExceptionHandler("/Error");
    // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
    app.UseHsts();

    //app.UseWebAssemblyDebugging();
}

app.UseHttpsRedirection();

app.UseStaticFiles();

app.UseRouting();

app.MapRazorComponents<App>()
    .AddServerRenderMode()
    .AddWebAssemblyRenderMode();

app.Run();
```

```c#
@page "/counter"
@attribute [RenderModeWebAssembly]

<PageTitle>Counter</PageTitle>

<h1>Counter</h1>

<p role="status">Current count: @currentCount</p>

<button class="btn btn-primary" @onclick="IncrementCount">Click me</button>

@code {
    private int currentCount = 0;

    private void IncrementCount()
    {
        currentCount++;
    }
}
```

**Comment by marinasundstrom** [comment]
Does anyone know how to solve this?

**Comment by maraf** [comment, marked as answer]
The new "web" template doesn't support WebAssembly yet https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-8-preview-5/#blazor

> Also, while some of the APIs for the Blazor WebAssembly render mode have been added in this preview (like the `[RenderModeWebAssembly]` attribute), support for running interactively on WebAssembly isn’t fully implemented yet. That will come in a future update too.

**Reply by marinasundstrom** [reply]
Yes. But sure it would work with the daily build of .NET 8 Preview 6? There are tests utilizing ``[RenderModeWebAssembly]`` so I assume that it would work in a dev build?

**Reply by maraf** [reply]
The only tests that I found are these two
- https://github.com/dotnet/aspnetcore/blob/8e2851f7d4ab42bd473fcd4489942de18d893312/src/Components/Endpoints/test/RazorComponentEndpointDataSourceTest.cs#L377
- https://github.com/dotnet/aspnetcore/blob/8e2851f7d4ab42bd473fcd4489942de18d893312/src/Components/Endpoints/test/TestComponents/InteractiveWithInteractiveChild.razor#L4 used in https://github.com/dotnet/aspnetcore/blob/8e2851f7d4ab42bd473fcd4489942de18d893312/src/Components/Endpoints/test/EndpointHtmlRendererTest.cs#L1126

But none of them really runs the app on WebAssembly.

**Reply by marinasundstrom** [reply]
Ok. I've seen the activity. Will wait and try again in a future build.
